packages/radix-ui-themes/src/components/grid.css

```diff
@@ -7,4 +7,10 @@
   justify-content: flex-start;
   grid-template-columns: minmax(0, 1fr);
   grid-template-rows: none;
+
+  &:where(.rt-r-gt) {
+    grid-template-columns: unset;
+    grid-template-rows: unset;
+    grid-template: var(--grid-template);
+  }
```

Comment:
- This should be a separate utility in `grid-template.css` wrapped in `@breakpoints`
- Are `grid-template-columns` and `grid-template-rows` needed? iirc `grid-template` shorthand would overwrite the longhand values if they are present, wouldn't it?

Reply:
Good thought, fixed.